uui/components/pickers/PickerInput.tsx

```diff
@@ -80,7 +80,7 @@ export class PickerInput<TItem, TId> extends PickerInputBase<TItem, TId, PickerI
     }
 
     renderTarget(targetProps: IDropdownToggler & PickerTogglerProps<TItem, TId>) {
-        const renderTarget = this.props.renderToggler || (props => <PickerToggler { ...props } />);
+        const renderTarget = this.props.renderToggler || (props => <PickerToggler { ...props } cx={ this.props.inputCx }/>);
```

Comment:
let's add inputCx to targetProps parameter, that it will also passed to the renderToggler

Reply:
done